server/parameters/config.rs

```diff
@@ -28,14 +28,33 @@ impl Config {
             .map(|path| path.parent().unwrap().to_path_buf())
             .unwrap_or(std::env::current_dir().unwrap());
         Self {
-            server: ServerConfig { address: SocketAddr::from_str("0.0.0.0:1729").unwrap() },
+            server: ServerConfig {
+                address: SocketAddr::from_str("0.0.0.0:1729").unwrap(),
```

Comment:
Goes to `constants.rs` for DEFAULT_ADDRESS or something

Reply:
Fixed in 099a56db95553ad20099ee985f464cbdb1923ee5.